tests/test_pdb_hook.py

```diff
@@ -0,0 +1,42 @@
+import pytest
+import os
+import tempfile
+from tools import run
+import json
+import platform
+
+
+@pytest.fixture(autouse=True)
+def conan_test():
+    old_env = dict(os.environ)
+    env_vars = {'CONAN_HOME': tempfile.mkdtemp(suffix='conans')}
+    os.environ.update(env_vars)
+    current = tempfile.mkdtemp(suffix='conans')
+    cwd = os.getcwd()
+    os.chdir(current)
+    try:
+        yield
+    finally:
+        os.chdir(cwd)
+        os.environ.clear()
+        os.environ.update(old_env)
+
+@pytest.mark.skipif(platform.system() != "Windows", reason="Only for Windows with msvc")
+def test_copy_pdb_hook():
+    repo = os.path.join(os.path.dirname(__file__), "..")
+    run(f'conan config install {repo}')
+    conan_home = run('conan config home').strip()
+    hooks_path = os.path.join(conan_home, 'extensions', 'hooks')
+    run(fr'ren {hooks_path}\_hook_copy_pdbs_to_package.py hook_copy_pdbs_to_package.py')
+    run('conan profile detect')
+    run('conan new cmake_lib -d name=lib -d version=1.0')
+    out = run('conan create . -s build_type=Debug -o "*:shared=True"')
```

Comment:
No need to launch the test_package
```suggestion
    out = run('conan create . -s build_type=Debug -o "*:shared=True" -tf=""')
```